.clang-format

```diff
@@ -1,11 +1,14 @@
+StatementMacros: ['UPROPERTY', 'UFUNCTION', 'UCLASS', 'USTRUCT', 'UENUM', 'UINTERFACE', 'GENERATED_BODY']
 Language: Cpp
 BasedOnStyle: LLVM
+
 AccessModifierOffset: -4
 AlignEscapedNewlines: Left
-AllowShortEnumsOnASingleLine: false
 AllowShortBlocksOnASingleLine: Empty
+AllowShortEnumsOnASingleLine: false
 AllowShortFunctionsOnASingleLine: Inline
-BraceWrapping: 
+AllowShortLambdasOnASingleLine: None

```

Comment:
`AllowShortLambdasOnASingleLine=Inline` is seemingly bugged. `AllowShortFunctionsOnASingleLine=Inline` allows function declaration and definition all on one line:
```
int Example() { return 42; }
```
Whereas the equivalent for lambdas gives this:
```
auto const Example = []()
    { return 42; }
```
This seems to be because `BeforeLambdaBody` in `BraceWrapping` takes precedence over `AllowShortLambdasOnASingleLine`, when it shouldn't in order to stay consistent with `AllowShortFunctionsOnASingleLine`. Disabling `BeforeLambdaBody` allows non-Allman style brace wrapping for lambdas so either way you get screwed.